docs/developers/bitesize/ci/CaaS_DEV_CD.svg

```diff
@@ -0,0 +1 @@
sUq8HmhYLsHT2IM8HIFjKA9R/wTI+OKipanQY5Lb0BGX7urB4JbBgGIbOHneWnjuUd3uwGMwduWZQAQQWg+J6smsElie3PoO5I7AYFNeTXSOwPLn1GcwdgcWguJ7sGoHlya3PYO4ILAbF9WTXCCxPbn0Gc0dgMSiuJ7tGYHly6zOYOwKLQXE92TUCy5Nbn8HcEVgMiuvJrhFYntz6DOaOwGJQXE92jcDy5NZnMHcEFoPierJrBJYntz6DuSOwGBTXk10jsDy59RnMHYHFoLie7BqB5cmtz2DuCCwGxfVk1wgsT259BnNHYDEorie7RmB5cuszmDsCi0FxPdn1/wAfcDU8l0CydgAAAABJRU5ErkJggg==" preserveAspectRatio="none"/><image width="1" height="1" id="ae" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJQAAACWCAYAAAA49KHfAAAgAElEQVR4XuR9B3wcxfX/d3b3+p2kU7esZlvu3XLFFNNL6L2akJDQQkL5QQIBkkCAkIQQQknomFBtugPGVBtjy00usrFxldV7uTtd3935f97snSzJKifblOT/8hFydHu7M2++8/qbZfj/kxgAVlxcLKuqPIxLPE+SlCwO5OtcH8/AssH5EM54GgNL5pzburCJMyDAgTYGNINJ1WB6tcTMW1VdbVJ0Vs2YtK+oKKdh0aJFOgD+/xOLibH/vxAbN26cw2RKTmayOk9n8rWKosxVVZWBc3B+eNadMQZJkghHUZ2zRbKsPa6FQnsCgaGe3buXRP7XAfa/DihpypR5SVwKncSAn3GGIs55LgClK4AISvRDzFAYYBJbTOAMnP7XhUuM0yeM03/ozzo4VABqDI8Epa5EAOMcHbIiV+i6vl7StCciEcf2bduW+f8XwfW/CCipuPgEV1T3niZJykmarp3PGHNyTtpnv/4xM0DiQIbCUKhIPFdiLFsGT5UYkhljyTLjLgA2RYKFcwEeIo2BBznnHRqkAIPernHm0TmaNaBW56hROfbqHH5NZwS0KAe6goykF+d8uyzJS/So9k40atu0bduyjv8VNfE/A6ji4mJTlFmHc007W5JxJziSdH0/iGhRJcb4CEVCgYmxmVZJn6tIcHMukUFEi6/FpNRgFzcu3WQAJgZYGcMujaMkrGFjlKNW46hUdSEB6RlxgBG4dF3/2mSS744E9FVbx+Y3Y9EiuuS/lv4XAMUmz5g7DtB+x7h0qqapzvhqRDiQJjMMN8n8NAvDiWaJhbuA5vBYTX2vfZy59JyQxPCWX9W/DOusWtNJcjECYIx0SZbawfmjkYD6zPbtpXX/rYj6rwbUlBlHzGCc36zq+o8YkER2EdkyLplhhCLxk60Sm6AwDJWYsJEMefX9EdlmBOhKjfOvIjpWhHVUqLpYA+F2kmKVpL3g+hsqjz6/bcOG3d/faA/uyf99gPr976Wp7y0ZA0X+hc755VzXXQQkAotbYphrkXC8VebTFIlFYn8/ONZ8u98iG87PgZIo54sDKnaqnAU5B0ktoQq5Xikx9g8elf9dVraq8dsdzeG7+38boNjE6bNvYzp/KO6lEZA0DlzhVHC+XdZTOKTo4ePPd3KnZAnYqIK/HFDxeVBlZIPRwsQM+B2AdFPZhpKPvpPBHOJD/msANXH6rNNkSL/XdX0K59ykc8ApM5xlk/nZVpklx2bybdtFh8jvPr9Ohjo5BuU6528ENHwa0pgldjVjUgcYX2yC+pvS0tKqH3K44QcPqHEzZmQruvJLcP1XnHN7PGY0z6bwy6wSGyYz4Tn1Rc1kmDPDRvkhUFdDvbfx0OckYctUjhc6otihGmpQxLPA94LJd2tBz5vbtm2jIOkPjn4ofO6NMdKkWbNyoGIxOEgqCTspR2b87hQzG8OAgThK4GsDgxsUh/xhEBnln0Q5zjSxfp0EI2jK8GFE5094I4zmHjPcaVrPqiHbLT/E+NUPhc8HrPbkqbOu0IFHAe6mD/1cxy0uC060SHCyxBJkjTqHTweKFMPL+yEQjWOTyjFKZnAkwH26vl7neMKvY21YFdKKsCZJ8nbOoueVrV+/44cwr/gYEpjSdzvcwnmF1mRfNtlKt1ImhAzuQrOEm5wmjOsSuBloVGYAD/g13OiQEbdFyLP6PolCGgSQjVEdrZziYv2r665jJTX4YUjnz3VEWafTwVAHhl9tKV276PucV9dnf88s7s6GiROPdDNT9FkOnEsJMLKNjrHKuMUhwy5siMTpa5XjqY4oHnObRU6OVMY9PrrjYO6S+PMGupKeOt8mY7TC4IWOh7wa7ksydUvLDHQPWqxyHfx33ijqVZ2RIc8IkxzXlG1c+8IPINT2gzEtMG7OnFQlopdwnY8SMh3A7SlmfrzSmUYbiN+dn5Mgez6sw9zmwcVZbpHFJSid0Rz+nuBkwPjeJBNmmiUEmIarmqO4xWXGLHPPdPLA0wxy4P4Ola8LazFQMZ0x052bS7/6y/cNqh+EhBo3bka2bJWWgvNJxM4kieE2l4nPNDFK7g8aBFQxcHFrBJf4vDi5IA0KN1RLHFBk4IdV9TvZTbIkQ5GlboDyBiN43BtCxGHHg05FRM8TJVow4gmFGF4ManyRX42BSgKT8DeFRym08L2F4r53QI0bNyfVZOUrdK6PI6amSQz3JJv4aJmJXNfXOjBCApQEOU4Teies498dEfysw4t5+emwMqkboFKSXHj+iUcQDg9mKRMcQI/L3lr8Id5874NugPIEwvi4rh2L01LxTIqp08ZL5Ak0YgqF5IokNPCGsKtUkRVgBC2Gf5kQ/eX3BarvFVBjZs5Ms2hKja6rwm52SAwvpZq5XZgGwBoNmKYwKl5KhNfiGvreYx0qytv9uNDvx/jCDDgtpm6Acqck4aUXn0Uo9O0DauGb72DRm293B1QwIgD1r6Rk/Nyh4GKbnLAUpk32UVTH8YoEKVa7tTz
```

Comment:
There's also a slightly ominous `X` at the end of the "developer" timeline but not any of the others, is that deliberate?

Reply:
This might as well be a binary file. LucidChart generated it and I'm not manually touching it.

Hopefully lucid chart didn't insert any malicious code in the svg 🕶

Reply:
words can not describe how much scrolling i just had to do

words also can not describe how stupid i am for not giving up sooner

Reply:
In this case, I meant in the rendered diagram: https://cromwell.readthedocs.io/en/ks_cicd250/developers/bitesize/ci/CICD/

Reply:
> There's also a slightly ominous X at the end of the "developer" timeline but not any of the others, is that deliberate?

Oops. I copy/pasted that "X" from the lucidchart example. It shouldn't be there, as it does ominously indicate the [termination of the developer](https://creately.com/blog/diagrams/sequence-diagram-tutorial/).